### PR TITLE
README.md: howto execute development versions for non-devs

### PR DESCRIPTION
**Pull request by Krysztophe**
1) Installation instructions of the dev version for people with low python skills, like me (thanks @blogh).

2) I've added the name of the packages from the PGDG. The one from Debian is not so obvious.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,20 +8,53 @@ Command line tool for PostgreSQL server activity monitoring.
...
+    $ ~/.local/bin/pg_activity
+
+Installation from the git repository
+------------------------------------
+
+This is only necessary to test development versions. First, clone the repository:
+
+    $ git clone https://github.com/dalibo/pg_activity.git
+
+Change the branch if necessary. Then create an dedicated environment,
```
Comment (blogh) [inline README.md:44]:
```suggestion
Change the branch if necessary. Then create a dedicated environment,
```

File: README.md [hunk 2]
```diff
@@ -8,20 +8,53 @@ Command line tool for PostgreSQL server activity monitoring.
...
+install dependencies and then pg\_activity from the repo:
```
Comment (blogh) [inline README.md:45]:
```suggestion
install pg_activity and it's dependencies. Finally, run pg\_activity from the 
virtual environment:
```

File: README.md [hunk 3]
```diff
@@ -8,20 +8,53 @@ Command line tool for PostgreSQL server activity monitoring.
...
+---------------------
 
 Alternatively, pg\_activity can be installed using pip on Python 3.6 or later
 along with psycopg2 (version 2.8 or higher):
 
     $ python3 -m pip install pg_activity psycopg2-binary
 
+Then launch:
+
+    $ ~/.local/bin/pg_activity
```
Comment (blogh) [inline README.md:35]: Not sure the info is needed since calling pg_activity works just fine as is.

Comment (blogh) [reply README.md:35]: Thanks for  the PR !

File: README.md (hunk 2 above)
Comment (Krysztophe) [reply README.md:45]: Dependencies first, isn'it ?

And « its », not « it's »

File: README.md (hunk 3 above)
Comment (Krysztophe) [reply README.md:35]: When installing through pip  with the postgres user on Ubuntu, I have this:

```
  WARNING: The script pg_activity is installed in '/var/lib/postgresql/.local/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
```

I suppose that `.local/bin` is not in the `$PATH` of system users.

File: README.md [hunk 4]
```diff
@@ -8,20 +8,53 @@ Command line tool for PostgreSQL server activity monitoring.
...
+    $ git clone https://github.com/dalibo/pg_activity.git
+
+Change the branch if necessary. Then create a dedicated environment,
+install dependencies and then pg\_activity from the repo:
+
+    $ python3 -m venv MYENV
+    $ source MYENV/bin/activate
+    $ python3 -m pip install psycopg2-binary
+    $ python3 -m pip install pg_activity/
+    $ pg_activity
```
Comment (dlax) [inline README.md:51]: I'd recommended the following:
```console
$ cd pg_activity
$ python3 -m venv .venv
$ . .venv/bin/activate
(.venv) $ pip install . psycopg2-binary
(.venv) $ pg_activity
```

- `.venv` is an usual name for virtualenvs
- it's quite common to put the virtualenv inside the source tree
- when the venv is activated, one can use `pip` directly
- and install from current directory

Comment (Krysztophe) [reply README.md]: Thanks!  PR updated

**Review by dlax — APPROVED** [review]

**Pull request merged**